symphony-utility/src/main/java/org/symphonyoss/symphony/bots/utility/message/SymMessageBuilder.java

```diff
@@ -21,21 +23,43 @@ private SymMessageBuilder(String message) {
     this.message = message;
   }
 
+  /**
+   * Define message content. It should follow the MessageML v2 spec.
+   *
+   * @param message Message content
+   * @return Builder class
+   */
   public static SymMessageBuilder message(String message) {
     SymMessageBuilder builder = new SymMessageBuilder(message);
     return builder;
   }
 
+  /**
+   * Define entity JSON.
```

Comment:
Check if this sounds better:
_Define entity data. It should be a serialized JSON entity_